com.microsoft.mrtk.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -79,7 +79,29 @@ public Transform HostTransform
 
                 return hostTransform;
             }
-            set => hostTransform = value;
+            set
+            {
+                if (interactorsSelecting.Count == 0)
```

Comment:
Should this be `!=` to check for an active manipulation?

Reply:
ooooops

Reply:
Fixed